enemies/bosses/scenarios.json

```diff
@@ -70,6 +70,81 @@
       "incomingDamage": [],
       "note": "Botwoon 2 can not hit Samus while she is in the bottom left corner of the room."
     },
+    {
+      "id": 3,
+      "name": "Reverse Botwoon",
+      "boss": "Botwoon 1",
+      "requires": [],
+      "bossDodgeRate": 50,
+      "excludedWeapons": [
+        "Missile",
+        "Charge+Wave",
+        "Charge+Ice+Wave",
+        "Charge+Wave+Spazer",
+        "Charge+Ice+Wave+Spazer",
+        "Charge+Wave+Plasma",
+        "Charge+Ice+Wave+Plasma"
+      ],
+      "damageWindows": [
+        {
+          "name": "Magic Pixel Beam Fight",
+          "windowPercent": 15,
+          "requires": [
+            "Charge",
+            "canPixelPreciseTrick"
+          ],
+          "note": [
+            "Requires standing in a certain pixel range away from the wall.",
```

Comment:
Is this a pixel range or a single pixel? The new tech sounds like a single pixel. (Is it 1 pixel for shooting diagonal up, and a different one for diagonal down?)